lib/generators/solidus_subscriptions/install/templates/initializer.rb

```diff
@@ -96,4 +96,18 @@
   # with new subscription cycles by clearing any past failed installment when a new one is created
 
   # config.clear_past_installments = true
+
+  # ==================================== Custom Order Creation =====================================
+  #
+  # This settings allows the customization of the creation of each Installment Order and / or
+  # adding additional attributes to the Order
+  #
+  # config.order_creation_class = 'SolidusSubscriptions::OrderCreator'
```

Comment:
Can we call this `order_creator_class` instead of `order_creation_class`? That's the naming standard we use for most service objects in Solidus.

Reply:
Sure :)